Update `People/Beatmap_Nominators/Beatmap_Veto` to reflect the veto rework (#14388)
ppy/osu-wiki: wiki/People/Beatmap_Nominators/Beatmap_Veto/en.md

The *beatmap veto* allows a [Beatmap Nominator](/wiki/People/Beatmap_Nominators) to withhold a beatmap from ranking if they feel there are significant issues regarding beatmap quality which make it unfit for the [Ranked](/wiki/Beatmap/Category#ranked) section. Seeking further discussion or clarification over any kind of quality issues which they feel need to be addressed before the map can proceed with [Qualification](/wiki/Beatmap_ranking_procedure#qualification) is one of their main responsibilities.

> Any Beatmap Nominator (BN) or member of the Nomination Assessment Team may place a beatmap veto, provided they are formally qualified for the game mode(s) in question, as stated in the [Beatmap Nominator Rules](/wiki/People/Beatmap_Nominators/Rules). Each beatmap veto must be discussed by both parties — the mapper as well as the Nominator placing the beatmap veto — in an attempt to reach an agreed compromise.

Reviewer: ```suggestion
Any Beatmap Nominator (BN) or member of the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (NAT) may place a beatmap veto, provided they are formally qualified for the game mode(s) in question, as stated in the [Beatmap Nominator Rules](/wiki/People/Beatmap_Nominators/Rules). Each beatmap veto must be discussed in an attempt to reach an agreed compromise.
```

> In cases where discussion does not take place or is inconclusive, and either the mapper or vetoing Nominator requests it, members of the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (NAT) will initiate and oversee a mediation to either uphold or dismiss the veto. The mediation vote is held anonymously among the Beatmap Nominators and members of the NAT of a given game mode. Once voting has completed, the reasoning from each participant is revealed along with the mediation result in the beatmap discussion.

Reviewer: ```suggestion
In cases where a discussion is inconclusive, the beatmap host or vetoers may request a veto go to a mediation vote. During mediation, a vote is held anonymously among the Beatmap Nominators and members of the NAT of a given game mode. Once voting has completed, the reasoning from each participant is revealed along with the mediation result in the beatmap discussion.
```

## Initiating a beatmap veto

> As a Beatmap Nominator, you can initiate the process of vetoing a beatmap by clicking `Submit anonymous veto` on [the BN Management website's Vetoes page](https://bn.mappersguild.com/vetoes) and filling out the prompts to submit a pending veto. This includes a thorough explanation of the reason(s) for the veto. NAT members may submit pending vetoes in the same way. Pending vetoes are only visible to BN and NAT members.

Reviewer: ```suggestion
As a Beatmap Nominator, you can initiate the process of vetoing a beatmap by clicking `Submit anonymous veto` on [the BN Management website's Vetoes page](https://bn.mappersguild.com/vetoes) and filling out the prompts to submit a veto. This includes a thorough explanation of the reason(s) for the veto. NAT members may submit vetoes in the same way. Until additional BNs vouch for a veto, a veto will only be visible to BNs and NAT members.
```

Reviewer: We don't use the language of "pending", so keeping it consistent. It will be worth considering if it should have its own name down the line for clarification though.

> Once this pending veto is submitted, two Beatmap Nominators must vouch for it in order to veto the beatmap. Vouching for a veto is an indication of support for the veto, and is considered equivalent to submitting a veto writeup. NAT members **cannot** vouch for vetoes.

Reviewer: ```suggestion
Once this veto is submitted, it will be available for Beatmap Nominators in the affected game mode(s) to vouch. Vouching for a veto is an indication of support for the veto, and is considered equivalent to submitting a veto writeup. NAT members **cannot** vouch for vetoes.
```

> Once a veto writeup has been vouched for by two Beatmap Nominators, the relevant beatmap is considered vetoed, and the [Nomination Assessment Team bot](https://osu.ppy.sh/users/6616586) will post to the affected map's discussion about it, stating the reason(s) for the veto.

Reviewer: ```suggestion
If a veto writeup obtains vouches from two additional Beatmap Nominators, the veto reason will be made public, and the relevant beatmap is considered vetoed. The Nomination Assessment Team will post to the affected map's discussion stating the reason(s) for the veto.

If a veto does not obtain two vouches before a beatmap is Ranked, it will instead be dismissed.
```

Reviewer: 1. This process cannot be automated (website limitation) and we have been a bit inconsistent whether the bot account is used or not. Best to keep it general for now.

2. Added comment on dismissal due to lack of vouching

Author: Is it "to rank a beatmap" or "to Rank a beatmap" with a capital letter?

Reviewer: Updated the suggestion accordingly - but in "to rank" I usually see lowercase, while the category and status is capitalized.

## Discussion

> Once a beatmap is vetoed, the vetoers (users who submitted and vouched for the veto), as well as the beatmap's creator and any guest difficulty creators whose difficulties are affected, are invited to a private veto chatroom. Additionally, members of the NAT have access to view and delete messages in the chatroom in order to moderate it.

Reviewer: ```suggestion
Once a beatmap is vetoed, the vetoers (users who submitted and vouched for the veto), the beatmap host, the beatmap's nominators, and any guest contributors whose contributions are affected, are invited to a private veto chatroom. Additionally, members of the NAT have access to view and delete messages in the chatroom in order to moderate it.
```

Reviewer: Making the list comprehensive + changing guest mappers to contributors to cover edge cases such as storyboarders, hitsounders, skinners, etc.

Author: Is a capital-N "Nominator" in "the beatmap's Nominators" correct?

Reviewer: You know, I'm not sure because the wiki does not really reference a beatmap's specific nominators like this to my knowledge. I would think it is as it's still more of a title, but there's no consistent reference (one page I found uses both "Nominators" and "nominators") or style rule. 

beatmap's BNs or beatmap's Beatmap Nominators would not read well in this context either. Something like "involved beatmap nominators" would be potentially confusing since the vetoers are also involved beatmap nominators.

I guess wiki maintainers would be more suited to answer🤔

Reviewer 2: a drive-by comment: "nominators" (that is, people who nominate) would be more fitting, similarly I hope no one says "difficulties from Guest Mappers, verified by Modders"

> All users in the private veto chatroom are anonymous to begin with<!--though I think this will be changed very soon -->, but a user may choose to reveal their identity at any time.

Reviewer: ```suggestion
The vetoers in the private veto chatroom are anonymous to begin with, but a user may choose to reveal their identity at any time.
```

> The vetoers are expected to discuss the reasons for the veto with the mapper(s) and try to reach a compromise. If the mapper thinks they have made sufficient changes to resolve the vetoers' concerns, they may initiate a vote to dismiss the veto. The vetoers will be prompted to vote to uphold or dismiss the veto. If two of the three vetoers vote to dismiss the veto, the veto is dismissed, and the beatmap may be ranked.

Reviewer: ```suggestion
The vetoers are expected to discuss the reasons for the veto with the mapper(s) and try to reach a compromise. If the mapper or vetoers thinks the concern has been sufficiently addressed through discussion or changes made to the beatmap, they may initiate a vote to dismiss the veto. The vetoers will be prompted to vote to uphold or dismiss the veto. If two of the three vetoers vote to dismiss the veto, the veto is dismissed, and the beatmap may be ranked.
```

Two Beatmap Nominators for the osu!taiko mode vouch for the veto. The map is therefore considered vetoed, and a beatmap discussion post containing the reasons for the veto is created by a member of the NAT. A chatroom is created, and Oxidisation, Drum-Defender, the beatmap's nominators as well as the vetoing and vouching Beatmap Nominators are invited to discuss the veto.

After a few days of discussion, Oxidisation believes Drum-Defender has changed the beatmap enough to be acceptable to all parties, so they initiate a vote to dismiss the veto. The two vouching Beatmap Nominators vote to dismiss the veto, so the veto is dismissed.

### Example 2

A member of the Nomination Assessment Team believes a Qualified beatmap by contracause uses a song which is extremely poorly composed to the point of being unfit for the Ranked section. The NAT member writes up a veto with this reasoning and submits it via the BN Management website.

> Two Beatmap Nominators vouch for the veto. The map is therefore considered vetoed, and a member of the NAT posts a suggestion on the affected beatmap containing the reason for the veto, thereby temporarily blocking it from being Ranked. A chatroom is created, and contracause, the beatmap's nominators and the vetoing and vouching Beatmap Nominators are invited to go to the veto.

Author: ```suggestion
Two Beatmap Nominators vouch for the veto. The map is therefore considered vetoed, and a member of the NAT posts a suggestion on the affected beatmap containing the reason for the veto, thereby temporarily blocking it from being Ranked. A chatroom is created, and contracause, the beatmap's nominators and the vetoing and vouching Beatmap Nominators are invited to discuss the veto.
```
